test/javabind.cpp

```diff
@@ -298,10 +299,11 @@ struct StaticSample
         return
             [](const std::string& str)
             {
-                char* ptr;
-                double val = strtod(str.data(), &ptr);
-                if (ptr != str.data() + str.size()) {
-                    throw std::runtime_error("strtod");
+                std::istringstream iss(str);
+                double val;
+                iss >> val;
+                if (iss.fail() || !iss.eof()) {
+                    throw std::runtime_error("istringstream");
```

Comment:
I changed this to use std::istringstream because strtod is locale dependent (e.g. on my system the tests would fail because it expects a , intead of a . in a fractional number)